org.fsnebula.Knossos.yml

```
@@ -0,0 +1,68 @@
+app-id: org.fsnebula.Knossos
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
```

```suggestion
runtime-version: '5.15-22.08'
```
current version is this.

line 6 also